modelon/impact/client/entities/workspace.py

```diff
@@ -545,3 +629,117 @@ class object.
             self._model_exe_sal,
             self._exp_sal,
         )
+
+    def _refresh(self):
```

Comment:
I don't think we should have these refresh methods that will result in mutation of the data depending on things happening in other places instead of having an explicit update made by the user.